connection.go

```diff
@@ -100,8 +100,8 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 			conn.Addr(), err)
 	case LogUnexpectedResultId:
 		header := v[0].(Header)
-		log.Printf("tarantool: connection %s got unexpected resultId (%d) in response",
-			conn.Addr(), header.RequestId)
+		log.Printf("tarantool: connection %s got unexpected request ID (%d) in response %s",
+			conn.Addr(), header.RequestId, "(probably canceled request)")
```

Comment:
If it's just a deal of the line length:
```suggestion
		log.Printf("tarantool: connection %s got unexpected request ID (%d) in response " +
			"(probably cancelled request)",
			conn.Addr(), header.RequestId)
```